remotespark/livyclientlib/livyclient.py

```diff
@@ -10,17 +10,18 @@ class LivyClient(object):
 
     def __init__(self, session):
         self._session = session
+        self._session.create_sql_context()
```

Comment:
It is kind of expensive and thought about it for a while. Here's my thought process: if we don't do this and the user runs code that creates a sqlContext, next time that he runs a -s True (sql query), he won't find the tables he defined on the context because we would have overridden it. By virtue of instantiating it here, we would provide the guarantee that as long as he doesn't redefine sc or sqlContext, both contexts are available and should work.